src/Diagnostics.Logger/HeaderConstants.cs

```diff
@@ -50,6 +50,11 @@ public static class HeaderConstants
         /// </summary>
         public static readonly string GeomasterNameHeader = "x-ms-geomaster-name";
 
+        /// <summary>
+        /// Location header name.
+        /// </summary>
+        public static readonly string LocationHeader = "x-ms-location";
```

Comment:
Applens (internal CSS accessing the diagnostics) won't be able to pass location details for all resource types. I think we should remove this. This header will be present for ARM, however the same codebase is shared for internal and external so it is best not to take dependency on this header.